pkg/katautils/config.go

```diff
@@ -250,6 +251,23 @@ func (h hypervisor) firmware() (string, error) {
 	return ResolvePath(p)
 }
 
+func (h hypervisor) PFlash() ([]string, error) {
+	pflashes := h.PFlashList
+
+	if len(pflashes) == 0 {
+		return []string{}, nil
+	}
+
+	for _, pflash := range pflashes {
+		_, err := ResolvePath(pflash)
+		if err != nil {
+			return []string{}, fmt.Errorf("pflash path doesn't exist: %v", pflash)
```

Comment:
Nit: use `%s` instead of `%v`, since we known `pflash` has a `string` type

Reply:
yeah, it's better